nats/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+nats.uri=localhost:4222
```

Comment:
why this property in `nats` subproject?

I expect all configuration properties are located in root subproject